cmd/juju/backups/doc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package backups provides the juju backups command.
+// Backup of juju's state is a critical feature, not only for juju users
+// but for use inside juju itself. This is likewise the case for the
+// ability to restore previous backups.
+
+// Backing up juju state involves dumping the state database and copying all
+// files that are critical to juju's operation. All the files are bundled up
+// into an archive file. Effectively the archive represents a snapshot of juju state.
+
+// First of all, running juju commands will fail while the DB is
+// unavailable (already running services should not be affected). While
+// this period of time is rather short, we expect that it will grow over
+// time. Furthermore, the larger a model's state, the larger the
+// impact of this downtime. In the long term this makes it less than ideal
+// to run backup as often as it should be.
+
+// Secondly, state currently does not block for the backup process as a
+// whole. This means that if we dump the DB first, it may be out of date
+// by the time we finish gathering the state-related files. In practice
+// this isn't a big concern since we do not expect the files to change
+// during the interval that backup is running. However, we do backup some
+// log files, so there is a small chance they will differ from when backups
+// started.
```

Comment:
This can be one sentence, probably further down after the mechanics have been explained.